slides/git-intro/index.html

```diff
@@ -323,6 +352,45 @@
                     <p>Once they have reviewed the changes, they can either merge the request, and your feature (or fix) is now in master, or comment on changes you need to make before they'll merge it!</p>
                     <p>Once it's merged.... move on to the next feature!</p>
                 </section>
+
+                <section>
+                    <h2>Recap</h2>
+                    <pre style="width: 120%; margin-left: -10%; box-shadow: 0px 0px transparent;">
+                        <code id="recap" class="zsh" data-trim contenteditable>
+$ mkdir tech103
+$ cd tech103
+$ git init
+$ git clone https://gitlab.com/Tech103/Wizards/tech103.git
+$ git checkout master
+$ git checkout -b tech103branch
+$ git checkout hotfix_001
+$ git add index.html
+$ git add .
+$ git commit -m 'Adding new index.html and assets'
+$ git pull origin master
+$ git diff tech103branch master
+$ git push origin tech103branch
+
+                        </code>
+
+                        <!-- Code comments HINT: press 'u' to unhide -->
+                        <span id="0" class="code-comment">// Make directory called tech103</span>
+                        <span id="1" class="code-comment">// Change to tech103 directory</span>
+                        <span id="2" class="code-comment">// Initialise bare repository</span>
+                        <span id="3" class="code-comment">// Clone remote repository</span>
+                        <span id="4" class="code-comment">// Checkout branch from master</span>
+                        <span id="5" class="code-comment">// Checkout new branch from head and switch</span>
+                        <span id="6" class="code-comment">// Checkout specific branch</span>
+                        <span id="7" class="code-comment">// Stage index.html pre-commit</span>
+                        <span id="8" class="code-comment">// Stage all files in current directory</span>
+                        <span id="9" class="code-comment">// Commit with message</span>
+                        <span id="10" class="code-comment">// Pull master into current branch</span>
+                        <span id="11" class="code-comment">// View diff between branches</span>
+                        <span id="12" class="code-comment">// Push commits to remote branch</span>
+
```

Comment:
Hmm.... I may have to replicate the code highlighting with css, rather than use < code > for this particular bit to get them on the correct line.... highlight.js doesn't handle bash/zsh very well with git commands anyway, so no biggy. Leave it with me... 😄